lesson_06/expression/src/expression_calculator.ts

```diff
@@ -2,7 +2,25 @@ export class ExpressionCalculator {
   /** Returns the calculation of ((a + b) * c) / d^e */
   calculate(a: number, b: number, c: number, d: number, e: number): number {
     // Implement your code here to return the correct value.
-    return 0;
+    const sum = this.add(a, b);
+    const pro = this.multiply(sum, c);
```

Comment:
Prefer `product` over `pro`, `exponent` over `expo`, and `quotient` over `quo`. Spell out the words in your variable names unless the abbreviation is more commonly used (e.g. `countryUsa` instead of `countryUnitedStatesOfAmerica`)